easyLine.rb

```diff
@@ -0,0 +1,4 @@
+# Clculate the sum of the squares of the binomial coefficients on a given line with a function called easyline(n)
+def easyline(n)
+  n == 0 ? 1 : easyline(n-1)*(4*n-2)/n

```

Comment:
Use style guide https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md
You can use n.zero? instead  n == 0.